Encrypted Ink/Screens/AccountsListViewController.swift

```diff
@@ -169,6 +174,56 @@ class AccountsListViewController: NSViewController {
         }
     }
     
+    @objc private func didClickImportFromMetamask() {
+        guard var libraryURL = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first else { return }
+        while !libraryURL.path.hasSuffix("Library") {
+            libraryURL.deleteLastPathComponent()
+        }
+        
+        guard
+            let dirPath = "/Application Support/Google/Chrome/Default/Local Extension Settings/".addingPercentEncoding(withAllowedCharacters: .urlPathAllowed),
+            let dirURL = URL(string: "file:///" + libraryURL.path + dirPath)
+        else {
+            return
+        }
+        let metamaskDirName = "nkbihfbeogaeaoehlefnkodbefgpgknn"
+        
+        let openPanel = NSOpenPanel()
+        openPanel.directoryURL = dirURL
+        openPanel.message = "Import from MetaMask"
+        openPanel.prompt = "Import"
+        openPanel.allowedFileTypes = ["none"]
+        openPanel.allowsOtherFileTypes = false
+        openPanel.allowsMultipleSelection = false
+        openPanel.canChooseFiles = true
```

Comment:
Should we enable choosing files or maybe it'd be enough to enable choosing directories?